tvb_storage/tvb/storage/h5/file/files_helper.py

```diff
@@ -244,6 +244,12 @@ def get_allen_mouse_cache_folder(self, project_name):
         self.check_created(folder)
         return folder
 
+    def get_tumor_dataset_folder(self, project_name):
+        project_folder = self.get_project_folder(project_name)
```

Comment:
I suggest we download the dataset directly under the TVB folder, I think it should be available for import in any project once it is downloaded. What do you think?

Reply:
I agree.